libs/api/domains/financial-aid/src/lib/financialAid.module.ts

```diff
@@ -0,0 +1,36 @@
+import { Module, DynamicModule } from '@nestjs/common'
+import { FinancialAidResolver } from './financialAid.resolver'
+import { FinancialAidService } from './financialAid.service'
+import { ApplicationApi, Configuration } from '../../gen/fetch'
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+
+export interface Config {
+  baseApiUrl: string
+}
+
+@Module({})
+export class FinancialAidModule {
+  static register(config: Config): DynamicModule {
```

Comment:
Should use the new “@island.is/nest/config” system instead of the root environments.ts and `register` pattern.